src/main/scala/mesosphere/marathon/core/group/GroupManager.scala

```diff
@@ -8,10 +8,10 @@ import akka.stream.scaladsl.Source
 import mesosphere.marathon.core.instance.Instance
 import mesosphere.marathon.core.pod.PodDefinition
 import mesosphere.marathon.state.{ AppDefinition, Group, PathId, RootGroup, RunSpec, Timestamp }
-import mesosphere.marathon.upgrade.DeploymentPlan
 
 import scala.concurrent.Future
 import scala.collection.immutable.Seq
+import mesosphere.marathon.upgrade.DeploymentPlan
```

Comment:
unclear why this commit was moved. was hoping that the last comment I left related to imports would have sufficed for both moved statements. the whole point was to avoid changes to this file that were unnecessary.